pipelines/test/docs.yaml

```diff
@@ -0,0 +1,63 @@
+name: docs
+
+needs:
+  packages:
+    - apk-tools
+    - grep
+    - man-db
+    - texinfo
+
+pipeline:
+  - name: docs readability check
+    runs: |
+      # Get our doc package name, which is usually a sub package
+      doc_pkg=$(basename ${{targets.contextdir}})
```

Comment:
We should be able to replace this with ${{subpkg.name}} which is in melange 19.4.

Reply:
Sounds like there's an effort to fix these en masse, with a bunch of others using something similar.